opm/grid/common/ZoltanPartition.cpp

```diff
@@ -96,6 +97,7 @@ makeImportAndExportLists(const Dune::CpGrid& cpgrid,
         if (allowDistributedWells)
         {
             wellsOnProc = perforatingWellIndicesOnProc(parts, *wells, cpgrid);
+            // In case we want to
```

Comment:
Unfinished sentence here?

Otherwise, looks fine, can be merged.

Reply:
Yeah, that where I first started and then I changed my mind... Fixed.